store/json_test.go

```diff
@@ -155,12 +155,12 @@ func TestKeyValuePairsAreWrittenAndReadCorrectly(t *testing.T) {
 
 // test case for testing newjsonfilestore idempotent
 func TestNewJsonFileStoreIdempotent(t *testing.T) {
-	_, err := NewJsonFileStore(testLockFileName, processlock.NewMockFileLock(false))
+	_, err := NewJsonFileStore(testLockFileName, processlock.NewMockFileLock(false), nil)
```

Comment:
Should be testing the condition where it is not `nil`. In that case, `zap.logger` should be called.

Reply:
@paulyufan2  was this addressed?

Reply:
yes it's fixed